src/pages/Beta/Pool/AllPoolList/PoolCard/styleds.tsx

```diff
@@ -11,7 +11,7 @@ export const Panel = styled(Box)`
   justify-content: flex-start;
   padding: 20px;
   border-radius: 10px;
-  height: 295px;
+  height: 240px;
```

Comment:
what is reason to reduce this height ? based on Figma currently height needs to 295px to support all drawer with constant height

Reply:
This was requested for me in this task https://app.clickup.com/t/344duuc